projects/angular/src/forms/checkbox/STYLES.md

```diff
@@ -0,0 +1,24 @@
+Toggle Switch - Style API
+
+## CSS Custom Properties
+
+| CSS Custom Property                           | Description                                           |
+| --------------------------------------------- | ----------------------------------------------------- |
+| --clr-toggle-bg-color-off                     | Background color for enabled toggle that is off       |
+| --clr-toggle-bg-color-on                      | Background color for enabled toggle that is on        |
```

Comment:
```suggestion
| --clr-toggle-bg-color-off                     | Background color for toggle that is off       |
| --clr-toggle-bg-color-on                      | Background color for toggle that is on        |
```
I would remove the `enabled` part as that's the default state of a toggle and it's not needed.